frontend/src/Editor/LeftSidebar/SidebarDebugger.jsx

```diff
@@ -208,6 +218,22 @@ function ErrorLogsComponent({ errorProps, idx, darkMode }) {
             sortKeys={false}
           />
         </span>
+        {errorProps.kind === 'restapi' && (
+          <span>
+            <ReactJson
```

Comment:
Don't we need `collapseStringsAfterLength` here also?

![image](https://user-images.githubusercontent.com/20333972/147482817-2b85541e-532d-4a07-9c70-532f2ce6581d.png)